src/main/java/gregtech/api/enums/Materials.java

```diff
@@ -1570,25 +1571,6 @@ public Materials(int aMetaItemSubID, MaterialSet aIconSet, float aToolSpeed, int
 		else mAspects.addAll(aAspects);
 	}
 
-	/**
-	 * This is for keeping compatibility with addons mods (Such as TinkersGregworks etc) that looped over the old materials enum
-	 */
-	@Deprecated
-	public String name() {
-		return mName;
-	}
-
-	/**
-	 * This is for keeping compatibility with addons mods (Such as TinkersGregworks etc) that looped over the old materials enum
-	 */
-	@Deprecated
-	public static Materials valueOf(String aMaterialName) {
-		return getMaterialsMap().get(aMaterialName);
-	}
-
-	/**
-	 * This is for keeping compatibility with addons mods (Such as TinkersGregworks etc) that looped over the old materials enum
-	 */
```

Comment:
There is no addons for 1.10.2 for GT and these methods have their non deprecated analogs